setup.sh

```diff
@@ -375,6 +375,9 @@ function _post() {
         echo_info "You can now use the box command to manage swizzin features, e.g. \`box install nginx panel\`"
     fi
     echo_docs getting-started/box-basics
+    #
+    # Source the bash_completion installer
+    . /etc/swizzin/sources/functions/bash_completion
```

Comment:
```suggestion
    # run the bash_completion installer
    bash /etc/swizzin/sources/functions/bash_completion
```

Reply:
does this change improve or fix anything?

Reply:
https://superuser.com/questions/176783/what-is-the-difference-between-executing-a-bash-script-vs-sourcing-it

Reply:
That does not answer the question. Whether source or execute in a subshell makes not difference to this this script.

Unless there is a specific reason running this in a subshell is an improvement to just running it.